### ISSUE TITLE
Ability to change what is selected when there is multiple things on the cursor

### ISSUE THREAD

**Issue by evactouch** [studio-minus/ppg-bugs-and-requests#556]
**Feature description**
New mechanic that allows you to change what your selecting when you have your cursor hovered over multiple items, and will say what got selected in the bottom right corner.

Ex: You hover over a human's right (front for 2d) forearm, but it selects the bottom torso. Lets say the assigned key for this is H. So you press H, and it changes the selected item and puts a message in the bottom right corner saying "Left forearm selected." so you press it again, and it reselects, and says "Right forearm selected." and now when you rightclick/leftclick it will grab or pull up the context menu for the selected item.

**Feature purpose**
Reselects the next item that the cursor is hovered over when there is multiple items using a assigned key.

**Additional context**
Made this because shaking the human so his arms wouldn't be over the legs/torso to grab the arm annoyed me.

**Comment by mestiez** [comment]
yeah this would be a nice feature, but it needs to be more elegant than this. Blender, and most other software, does it by cycling through all objects under the cursor every time you click.
